internal/rpcserver/rpcserver.go

```diff
@@ -3319,14 +3319,15 @@ func handleGetTreasurySpendVotes(_ context.Context, s *Server, cmd interface{})
 			if !checkingMainChain {
 				continue
 			}
-			for _, block := range blocks {
-				if !chain.MainChainHasBlock(&block) {
+			for i := range blocks {
+				blockHash := &blocks[i]
+				if !chain.MainChainHasBlock(blockHash) {
```

Comment:
On the matter of consistency in regard to my comment above, this loop requires a pointer variable in more than one place but it seems to have a single pointer variable is ok here.

Reply:
See previous.